extension/lib/CommonStorage.jsm

```diff
@@ -0,0 +1,44 @@
+ /* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { utils: Cu } = Components;
+
+Cu.import("resource://gre/modules/XPCOMUtils.jsm");
+
+XPCOMUtils.defineLazyModuleGetter(
+  this, "NewsIndexedDB", "resource://pioneer-study-online-news/lib/NewsIndexedDB.jsm"
+);
+
+this.EXPORTED_SYMBOLS = ["CommonStorage"];
+
+
+this.CommonStorage = {
+  getStore() {
+    return NewsIndexedDB.getStore("common");
+  },
+
+  async clear() {
+    await this.getStore().clear();
+  },
+
+  async getAll() {
+    return await this.getStore().getAll();
+  },
+
+  async get(key) {
+    return await this.getStore().get(key);
+  },
+
+  async add(value, key) {
+    return await this.getStore().add(value, key);
```

Comment:
`add(value, key)` seems backwards to me. I'd expect `(key, value)`.

Reply:
this matches the indexed db API.

Reply:
i believe they do it this way because when you have a keypath specified you don't need to pass  a key in and key becomes an optional param.